test/lit.cfg

```diff
@@ -1214,7 +1233,7 @@ runtime_libs = {
     'fuzzer': 'fuzzer_runtime'
 }
 
-if run_ptrsize != "32":
+if run_ptrsize == '64' and 'libdispatch' in config.available_features:
```

Comment:
This still seems wacky. Why rule out TSan-without-libdispatch?

Reply:
TSan now requires libdispatch, i.e., the TSan runtime has a dependency on it. On MacOS we always have at least the system-provided libdispatch. On Linux, we need to make sure it's available.

In addition, we need to make sure that libdispatch header/swiftmodule/library for linking is available when the test itself uses it. This is what `%import-libdispatch` attempts to do.